baselines/polars/tpch/Makefile

```diff
@@ -0,0 +1,56 @@
+SHELL=/bin/bash
+PYTHON=.venv/bin/python
+
+.venv:
+	@python -m venv .venv
+	@.venv/bin/pip install -U pip
+	@.venv/bin/pip install --no-cache-dir -r requirements.txt
+
+clean-tpch-dbgen:
```

Comment:
We don't want to make tpch-dbgen again. We already have `tpch-kit` in the root directory which when build provides a `./dbgen` executable that is used to generate data.

For the `run` commands you can assume that the data is available in a directory specified by a path variable and that it doesn't need to be generated.

Reply:
It makes sense that we don't want to make tpch-dbgen again, but what in particular should I get rid of here? Removing either `.venv` or `clean_tpch_dbgen` will cause problems with running polars.